docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,241 @@
+*Edit the system environment variables* control panel.
+
+OpenSSL
+^^^^^^^
+
+You can download the latest OpenSSL version for Windows at this webpage_.
+Download and use the installer that fits your requirements.
+After installing, add the environment variable :code:`OPENSSL_ROOT_DIR` pointing to the installation root directory.
+
+For example:
+
+.. code-block:: bash
+
+   OPENSSL_ROOT_DIR=C:\Program Files\OpenSSL-Win64
+
+Gtest
+^^^^^
+
+.. note::
+
+    By default, *eProsima Fast DDS* does not compile tests.
+    You can activate them by adding the :code:`-DPERFORMANCE_TESTS=ON` flag when calling colcon_ or CMake_
+    (for details, see below).
+
+You can find information on how to install Gtest at this `link <https://github.com/google/googletest>`_.
+
+.. _colcon_installation_windows:
+
+Colcon Installation
+-------------------
+
+colcon_ is a command line tool to build sets of software packages.
+This section explains how to use it to compile easily *eProsima Fast DDS* and its dependencies.
+
+.. important::
+
+    You need to run colcon within a Visual Studio prompt. To do so, launch a *Developer Command Prompt* from the
+    search engine.
+
+You can then install the ROS2 development tools (colcon_ and vcstool_):
+
+.. code-block:: bash
+
+    pip install -U colcon-common-extensions vcstool
+
+and add the path to the :code:`vcs` executable to your :code:`PATH` from the
+*Edit the system environment variables* control panel.
+
+.. note::
+
+    If this fails due to an Environment Error, add the :code:`--user` flag to your installation.
+
+Now, create a colcon_ workspace and download the repos file that will be used to install *eProsima Fast DDS* and
+its dependencies:
+
+.. code-block:: bash
+
+    $ mkdir Fast-DDS-ws && cd Fast-DDS-ws
+    $ wget https://raw.githubusercontent.com/eProsima/Fast-RTPS/master/fastrtps.repos
+    $ mkdir src
+    $ vcs import src < fastrtps.repos
+
+Finally, use colcon_ to compile all software:
+
+.. code-block:: bash
+
+    colcon build
+
+To run an *eProsima Fast DDS* instance, you need to source the colcon overlay with the command:
+
+.. code-block:: bash
+
+    setup.bat
+
+.. important::
+
+    The sourcing of the local colcon overlay is required every time the colcon workspace is opened in a new shell
+    environment to run an *eProsima Fast DDS* instance.
+    As an alternative, you can add it permanently to your :code:`PATH`.
+
+.. note::
+
+    If you want to compile the examples, you will need to add the flag
+    :code:`--cmake-args "-DCOMPILE_EXAMPLES=ON"` when running :code:`colcon build`.
+    If you want to compile the performance tests, you will need to add the flag
+    :code:`--cmake-args "--DPERFORMANCE_TESTS=ON"` when running :code:`colcon build`.
+    For this step, you need Gtest_ as explained in the requirements_windows_sources_ section above.
+
+
+.. _cmake_installation_windows:
+
+CMake Installation
+-------------------
```

Comment:
I'm not sure if windows can take the path syntax outlined in this section

Reply:
let's discuss it.

Reply:
verify the ~/Fast-DDS in windows

Reply:
/ vs \? I think they are the same

Reply:
change all / to \